app/views/accounts/actions/_manage_personal_key.html.slim

```diff
@@ -1,4 +1,7 @@
-= link_to manage_personal_key_path do
-  span.hide
-    = t('account.items.personal_key')
-  = t('account.links.regenerate_personal_key')
+= button_to(t('account.links.regenerate_personal_key'), create_new_personal_key_path, method: :post,
+    class: 'btn btn-link ml1', form_class: 'inline-block')
+
+/ = link_to manage_personal_key_path do
+/   span.hide
+/     = t('account.items.personal_key')
+/   = t('account.links.regenerate_personal_key')
```

Comment:
The original link had a hidden span, presumably for accessibility purposes. I wasn't sure if we needed it now that we're using `button_to`. If we do need it, where would it go?

Reply:
Looks like `button_to` can take a block and we can put the `span.hide` inside of that: https://github.com/18F/identity-idp/blob/master/app/views/accounts/actions/_disable_totp.html.slim#L2